articles/flows/concepts/device-auth.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Device Authorization Flow
+description: Learn how the Device Authorization flow works and why you should use it for input-constrained devices, such as smart TVs and media consoles. For use with native apps.
+topics:
+  - input-constrained-devices
+  - device-flow
+  - api-authorization
+  - grants
+  - flows
+  - authorization
+  - mobile-apps
+  - desktop-apps
+  - native-apps
+contentType: concept
+useCase:
+  - add-login
+  - secure-api
+  - call-api
+---
+# Device Authorization Flow
+
+With input-constrained devices that connect to the internet, rather than authenticate the user directly, the device asks the user to go to a link on their computer or smartphone and authorize the device. This avoids a poor user experience for devices that do not have an easy way to enter text. To do this, device apps use the Device Authorization Flow (drafted in [OAuth 2.0](https://tools.ietf.org/html/draft-ietf-oauth-device-flow-15)), in which they pass along their Client ID to initiate the authorization process and get a token.
+
+## How it works
+
+The Device Authorization Flow contains two different paths; one occurs on the device requesting authorization and the other occurs in a browser. The browser flow path, wherein a device code is bound to the session in the browser, occurs in parallel to part of the device flow path.
+
+![Device Authorization Sequence](/media/articles/flows/concepts/auth-sequence-device-auth.png)
+
+### Device Flow
+
+1. The user starts the app on the device.
+2. The device app requests authorization from the Auth0 Authorization Server using its Client ID (**/oauth/device/code** endpoint).
+3. The Auth0 Authorization Server responds with a `device_code`, `user_code`, `verification_uri`, `verification_uri_complete` `expires_in` (lifetime in seconds for `device_code` and `user_code`), and polling `interval`.
+4.  The device app asks the user to activate using their computer or smartphone and displays the `verification_uri` and `user_code`. (Instead of the user code, the app could also display a QR code or shortened URL with embedded user code.)
+5. The device app begins polling your Auth0 Authorization Server for an Access Token (**/oauth/token** endpoint) using the time period specified by `interval`. The device app continues polling until either the user completes the browser flow path or the user code expires.
```

Comment:
based on our own mishaps in client implementation we should note that each interval should start counting after receipt of the last request's response to avoid `slow_down` errors due to network latency